Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "reqwest"
-version = "0.11.9" # remember to update html_root_url
+version = "0.11.9"                                    # remember to update html_root_url
```

Comment:
Included extra space?

Reply:
Seems that my toml autoformatter went a bit overboard.